airguardian-core/src/main/kotlin/systems/ajax/motrechko/airguardian/mapper/CoordiateMapper.kt

```diff
@@ -1,15 +1,16 @@
 package systems.ajax.motrechko.airguardian.mapper
 
+import com.google.type.LatLng as ProtoLatLng
```

Comment:
usually alias is needed when you want to import two classes with the same name. Here I don't see other LatLng class, so alias can be omitted